src/main/java/Hachi.java

```diff
@@ -0,0 +1,266 @@
+import java.util.Scanner;
+
+/**
+ * This program currently starts the chatbot with a greeting,
+ * awaits user input for commands relating to a to-do task manager
+ * and ends off the program with a goodbye message.
+ *
+ * @author clarencepohh
+ * @version 06/02/2024
+ */
+
+public class Hachi {
+
+    /**
+     * Prints a greeting to the user in the console
+     * with the bot's name, Hachi.
+     */
+
+    public static void printGreetingMessage() {
+        String logo = "._. ._.  ._____.  ._____.  ._. ._.  ._.\n"
+                + "| | | |  | ._. |  |  ___|  | | | |  | |\n"
+                + "| |_| |  | |_| |  | |      | |_| |  | |\n"
+                + "| ._. |  | ._. |  | |___   |  _  |  | |\n"
+                + "|_| |_|  |_| |_|  |_____|  |_| |_|  |_|\n";
+
+        System.out.println("Hey, Hachi Here!\n" + logo + "How can I assist you today?\n");
+        spacerInsert("medium", false);
+    }
+
+    /**
+     * Prints to the console a help message that details the commands
+     * the user can use for interacting with the chatbot.
+     *
+     */
+
+    private static void printHelpMessage() {
+        spacerInsert("medium", true);
+        System.out.println("You can use the following commands:");
+        System.out.println("    'list' to retrieve your current list of tasks,");
```

Comment:
just a little note because i was originally doing this too. instead of adding an actual tab you can use the command "\t" to add a tab, and using a lot less spaces (since there is a line limit!) :)

Reply:
Thanks for letting me know! I will implement this 👍